tests/integration/conftest.py

```diff
@@ -201,6 +212,32 @@ def clean_up(self):
         self.remove_keys_and_certificates()
 
 
+@pytest.fixture()
+def fixture_subman(request):
+    """
+    Fixture.
+    Set up the subscription manager on the system. Wrapper around SubscriptionManager class and its methods.
+    Can be parametrized to set the subscription manager to the stagecdn (needed for SCA Enabled accounts):
+    @pytest.mark.parametrize("fixture_subman", ["set_to_stage=true"], indirect=True)
+    """
+    subman = SubscriptionManager()
+
+    subman.set_up_requirements()
+
+    if hasattr(request, "param"):
+        set_to_stage = request.param
+        print(f">>> Using parametrized option requested in the fixture - {set_to_stage}.")
+        if set_to_stage == "set_to_stage=true":
+            subman.set_up_to_stagecdn()
+
+    # TODO: should we add register here?
+    # Resolve under the: https://issues.redhat.com/browse/RHELC-1134
```

Comment:
We don't need to pre-register here.
The intention behind this is to just set the baseurl to stage.
That cannot be done without having the subscription-manager installed prior.
The registration is handled by convert2rhel itself.

Reply:
Ack, I remove it